awscli/clidocs.py

```diff
@@ -153,6 +155,18 @@ def doc_synopsis_end(self, help_command, **kwargs):
     def doc_options_start(self, help_command, **kwargs):
         doc = help_command.doc
         doc.style.h2('Options')
+        for arg in help_command.arg_table:
+            shape = help_command.arg_table.get(arg).argument_model
+            if is_streaming_blob_type(shape):
```

Comment:
Instead of adding this snippet to the top of the `Options` section, would it be possible to add it as part of the description to the argument? I think that would make it easier to notice.